.github/workflows/docker.yml

```
@@ -11,11 +11,23 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform:
+        - linux/amd64
+        - linux/aarch64
```

```suggestion
        - linux/arm64
```

Unless I'm missing something, seems like this should be arm64

https://stackoverflow.com/questions/31851611/differences-between-arm64-and-aarch64

`arm64` is self-documenting whereas with `aarch64` I might leave an `arm` comment next to it. Seems like they'd be interchangeable, possible even that only `arm64` works:

https://github.com/docker/setup-qemu-action/runs/3505773201?check_suite_focus=true 

Platforms:

```
linux/amd64,linux/arm64,linux/riscv64,linux/ppc64le,linux/s390x,linux/386,linux/mips64le,linux/mips64,linux/arm/v7,linux/arm/v6
```

---

Good callout! I'm more familiar with the `aarch64` identifier but it looks like the more correct name would be [`arm64v8`](https://github.com/docker-library/official-images#multiple-architectures) and more verbosely/explicitly, [`linux/arm64/v8`](https://github.com/docker-library/bashbrew/blob/v0.1.2/architecture/oci-platform.go#L14-L27) (following a format of `:os/:architecture/:variant`).